Data-Structures/Linked-List/test/SinglyLinkedList.test.js

```diff
@@ -148,6 +148,9 @@ describe('SinglyLinkedList', () => {
 
     list.addFirst(30)
     expect(list.head()).toBe(30)
+
+    list.addFirst(0)
```

Comment:
Perhaps add a comment that this test is for a falsy head; consider also using a more explicit falsy head such as `false` instead.